README

```diff
@@ -1,5 +1,7 @@
+#NOT ANOTHER LOUSY CHESS GAME(NALCG)
```

Comment:
Use of full caps is unacceptable. Rejected. Make a new one with better documentation.